package.json

```diff
@@ -11,7 +11,12 @@
     "build:sass": "sass app/assets/stylesheets/application.scss:app/assets/builds/application.css --no-source-map --load-path=node_modules --load-path=app/assets/stylesheets",
     "build:postcss": "postcss app/assets/builds/application.css --dir app/assets/builds",
     "build:css": "yarn build:sass && yarn build:postcss",
-    "build": "yarn build:js && yarn build:css"
+    "build": "yarn build:js && yarn build:css",
+    "lint:docs:yml": "eslint docs/openapi --ext .yml --color",
```

Comment:
Before, we installed the [`eslint-plugin-yml`](https://github.com/ota-meshi/eslint-plugin-yml) to have some predefined rules for the yaml lint. I haven't checked if the default is ok for our needs. Could you help double-check it?

Reply:
Not sure if I understand your concern correctly or not.

Following the [configuration](https://github.com/ota-meshi/eslint-plugin-yml#configuration), we can add these configs that has been defined by default.

```ruby
plugin:yml/base #... Configuration to enable correct YAML parsing.
plugin:yml/recommended #... Above, plus rules to prevent errors or unintended behavior.
plugin:yml/standard #... Above, plus rules to enforce the common stylistic conventions.
plugin:yml/prettier #... Turn off rules that may conflict with [Prettier](https://prettier.io/).
```

Maybe `recommended` is suit for our needs. 😜